### ISSUE TITLE
Profits Today showing last 24 hours

### ISSUE THREAD

**Issue by martinsvahn** [taniman/profit-trailer#604]
Hello.

I just updated to 1.2.3.9 and running on Binance.

I noticed "Profits today" was not the same after updating as it were before.
Did something change in this version?

![image](https://user-images.githubusercontent.com/5543082/34684405-48f3d286-f4a5-11e7-8f01-6b38cae042b2.png)

This is problematic when you keep daily profits in a spreadsheet, as it is changing all the time now during the day.

I would like to have a setting to get it the way it was before, like 00:00 - now();

**Comment by YoJeff** [comment]
Not exactly sure what you are talking about here -- you mention today's profits but then show a screenshot of the sales log.

The offset (0D -1D) is realtime, relative to now.   The time portion is absolute.

**Comment by martinsvahn** [comment]
Well, a screenshot of the profit today is not going to give much info?

"Profits today" usually reset at 00:00 for me to 0.00000000 BTC, but now it just shows last 24 hours always.

In my opinion "Profits today" should be sell orders on 2018-01-08 only, and "Profits yesterday" only sell orders on 2018-01-07. Like it was before.

**Comment by YoJeff** [comment]
I've not seen this myself, nor any other reports.   My profits today is always different from profits yesterday -- and always resets around 0:00.   If you upgraded recently, make sure you restarted the bot -- and hard refresh (CTRL-F5) the gui, to force reload any script changes.

**Comment by martinsvahn** [comment]
OK, after further investigation, I did add all the sales manually and it is displaying a correct value.

But, before I stopped the bot and updated the jar file, I did copy the value and put it in my spreadsheet. 0.0143 btc, and when i started the bot again it was saying 0.0165 btc, pretty big jump.

So maybe the older version I was using were not displaying it properly?

Sorry, I will close this now. There were no issues after all
